src/features/student/views/StudentEducationActivitiesView/components/StudentDetailedActivityCard/StudentDetailedActivityCard.vue

```diff
@@ -0,0 +1,260 @@
+<script lang="ts" setup>
+import type { AvLocale } from '@/types/i18n.types'
+import { formatDateToLocaleString } from '@/common/utils'
+import { studentActivityRoute } from '@/features/student/routes'
+import { type ActivityDTO, ActivityStatus } from '@/types'
+import { AvBadge, AvCard, IconText, MDI_ICONS } from '@/ui'
+import { useI18n } from 'vue-i18n'
+
+const { activity } = defineProps<{ activity: ActivityDTO }>()
+const { name, startedActivityCount, totalActivityCount, skillCount, traceCount, status } = activity
```

Comment:
Voici le DTO genéré par le back 
```json
export interface AmsViewDTO {
  id: string
  title: string
  countSkills: number
  countTraces: number
  status: AmsStatus
  progress: AmsProgress
}
```


```suggestion
const { name, startedActivityCount, totalActivityCount, countSkills, countTraces, status } = activity
```